### DISCUSSION TITLE
if any someone died what will happen to his GitHub account? ,  we want any sign to point this person has died and her work is alive

### DISCUSSION THREAD

**Discussion by ghost** [community/community#19618] [Other Feature Feedback, Questions, & Ideas] [answered]
if any someone died what will happen to his GitHub account? , 
we want any sign to point this person has died and her work is alive

**Comment by franzliedke** [comment]
There are "Successor" settings in your [account settings](https://github.com/settings/admin). There is also [a documentation article](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/maintaining-ownership-continuity-of-your-personal-accounts-repositories) on this topic.

**Reply by melvincwng** [reply]
Thanks +1 👍🏻

**Reply by ghost** [reply]
okay if someone died and we now can't mak this setting to make some to manage his work ,
i just want to make a pointer or sign to this person who has died ....like Facebook
we announce this person died by reporting to this account and Facebook make it and write a sentence to remember this person

**Comment by D7EAD** [comment, marked as answer]
Hi!

In the event a user passes away, Github has a thorough Deceased User Policy that you can find [here](https://docs.github.com/en/site-policy/other-site-policies/github-deceased-user-policy). Have a look and hopefully that answers any lingering questions you have!

If I've managed to help you, kindly consider **Marking as Answer** so others with the same question can easily find the appropriate answer. Thanks!
